src/App.js

```diff
@@ -1,12 +1,21 @@
 import React, { Component } from 'react';
-import './App.css';
+import Profile from './Profile';
+import Movies from './Movies';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <h1>Movie List</h1>
-      </div>
+      <BrowserRouter>
+        <div className="router">
+          <Switch>
+            <Route exact path="/" component={Movies} />
+            <Route exact path="/profile/:id" component={Profile} />
```

Comment:
I would probably name this something other than `Profile`, that works well for our `ContactList`, but this domain something like `MovieDetail` would be more appropriate.